orc8r/cloud/deploy/terraform/orc8r-aws/cert-manager.tf

```diff
@@ -0,0 +1,75 @@
+resource "random_id" "cert_manager_route53_random_id" {
```

Comment:
Can you add an explanatory comment for why including the random ID -- is it to ensure uniqueness within an AWS account? If yes, then a 1/256 chance of a "bug" with 2 Orc8rs feels a bit high (?)

In a future PR perhaps we could tie this to the Orc8r UUID... did we end up getting that into the Terraform @arunuke ?

Reply:
Yes, Magma UUIDs are now a thing that can be set when deploying via terraform (default value is....'default') and passed on to be set as additional annotations across various services deployed.

Reply:
IAM Roles is a Global AWS service. If we have another Orc8r in a different region it can conflict with the same role name.
I am looking into the Magma UUID code and trying to update it here.

Reply:
So if we use Magma UUID the user will have to know about this feature and update it manually on the second orc8r deployment. But if we use Random ID then there is no need for it because terraform will take care of this.
What are your suggestions for this @hcgatewood and @arunuke?
https://github.com/magma/magma/blob/master/orc8r/cloud/deploy/terraform/orc8r-aws/variables.tf#L47

Reply:
I suppose the middle ground is we automatically create a uuid for the orc8r if the user doesn't provide one? That way we get the benefit of this random solution, but remove the probability of conflict. Thoughts?